src/design-system/styles/template/block-areas/index.njk

```diff
@@ -0,0 +1,131 @@
+---
+title: Block areas – Template
+
+stylesheets:
+- ./block-areas-annotate.css
+---
+
+{% extends "template.njk" %}
+
+{% set bodyClasses = "app-example-page" %}
+
+{% block head -%}
+  <meta name="robots" content="noindex, nofollow">
+
+  <!--[if !IE 8]><!-->
+    <link href="/{{ fingerprint['design-system/stylesheets/main.css'] }}" rel="stylesheet" media="all" />
+  <!--<![endif]-->
+  <!--[if lt IE 9]>
+    <link href="/{{ fingerprint['design-system/stylesheets/main-ie8.css'] }}" rel="stylesheet" media="all" />
+    <script src="/{{ fingerprint['design-system/javascripts/ie.js'] }}"></script>
+  <![endif]-->
+  <script src="/design-system/javascripts/vendor/modernizr.js"></script>
```

Comment:
We need to set `head`, `footer` etc here as they're being excluded in metalsmith? It might be good to leave a comment here indicating that.